compute/cker/include/cker/operation/RmsNorm.h

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __NNFW_CKER_RMS_NORM_H__
+#define __NNFW_CKER_RMS_NORM_H__
+
+#include "cker/Shape.h"
+#include "cker/Types.h"
+#include "cker/Utils.h"
+
+#include <cmath>
+
+namespace nnfw
+{
+namespace cker
+{
+
+inline void RmsNorm(const RmsNormParams &params, const Shape &input_shape, const float *input_data,
+                    const Shape &gamma_shape, const float *gamma_data, const Shape &beta_shape,
+                    const float *beta_data, const Shape &output_shape, float *output_data)
```

Comment:
Is it necessary `beta`? I guess it is `bias`. If so, as I understand, `beta` is not necessary and not used.

- For our purpose model, it does not have `bias`.
- `llama.cpp` and transformer from HF also do not have `beta`.

Reply:
> Is it necessary `beta`? I guess it is `bias`. If so, as I understand, `beta` is not necessary and not used.
> 
> * For our purpose model, it does not have `bias`.
> * `llama.cpp` and transformer from HF also do not have `beta`.

Yes, beta(bias) is not necessary for our model, now.
RMSNorm in llama.cpp didn't have beta, too.
But, there is beta(bias) in RMSNorm paper and
I could find 'offset' from the reference code written by RMSNorm author.(Biao Zhang)
https://github.com/bzhangGo/rmsnorm/blob/master/rmsnorm_tensorflow.py

Reply:
@glistening 
Please review this.

Reply:
I also checked paper before you wrote comment. I can only find in section 4.1 Invariance Analysis. As I understand, `bias` doesn't matter. re-centering can be ignored for the sake of efficiency w/o sacrificing accuracy much. It is the point of the paper as I understand. It may be the reason the author's code ( personally it does not more important than llama.cpp or our model. It is not the code for production, but for wrting paper ) also disable `bias` by default. Please find:

>     :param bias: whether use bias term for RMSNorm, disabled by
>        default because RMSNorm doesn't enforce re-centering invariance.

Reply:
Okay, then I'll remove bias. :)

Reply:
@ seockho-kim

> Okay, then I'll remove bias. :)

If circle models always don't have `beta(bias)` for RMSNorm op, please remove it from all other code using RMSNorm op.

Reply:
> @seockho-kim
> 
> > Okay, then I'll remove bias. :)
> 
> If circle models always don't have `beta(bias)` for RMSNorm op, please remove the related code from all other code using RMSNorm op.

Sure, I'll remove related codes step by step. :)